2022-06-07 00:07:40,633 ERROR: </usr/lib/python3.8/threading.py> <self._target(*self._args, **self._kwargs)>: [Errno 104] Connection reset by peer
